pkg/resources/create_test.go

```diff
@@ -143,72 +139,83 @@ func TestCreateResource(t *testing.T) {
 	tests := []struct {
 		name string
 		json []byte
-		want resourcev1.PipelineResource
+		want pipelinev1beta1.TaskRun
 	}{{
-		name: "PipelineResource without namespace",
-		json: json.RawMessage(`{"kind":"PipelineResource","apiVersion":"tekton.dev/v1alpha1","metadata":{"name":"my-pipelineresource","creationTimestamp":null,"labels":{"woriginal-label-1":"label-1"}},"spec":{"type":"","params":[{"name":"foo","value":"bar\r\nbaz"}]},"status":{}}`),
-		want: resourcev1.PipelineResource{
+		name: "TaskRun without namespace",
+		json: json.RawMessage(`{"kind":"TaskRun","apiVersion":"tekton.dev/v1alpha1","metadata":{"name":"my-taskrun","creationTimestamp":null,"labels":{"someLabel":"bar"}},"spec":{"serviceAccountName":"","taskRef":{"name":"my-task"}},"status":{"podName": ""}}`),
```

Comment:
As you are changing it to TaskRun can we also make sure to use v1beta1 or v1 versions  instead of v1alpha1

cc @dibyom